src/net.cpp

```diff
@@ -493,12 +493,13 @@ void CNode::SetAddrLocal(const CService& addrLocalIn) {
 
 #undef X
 #define X(name) stats.name = name
-void CNode::copyStats(CNodeStats &stats)
+void CNode::copyStats(CNodeStats &stats, std::vector<bool> m_asmap)
```

Comment:
Should this be taking `std::vector<bool>&` instead? That's potentially a pretty large vector to be copying.